lib/modules/datasource/cdnjs/index.spec.ts

```diff
@@ -16,7 +16,10 @@ describe('modules/datasource/cdnjs/index', () => {
     });
 
     it('throws for empty result', async () => {
-      httpMock.scope(baseUrl).get(pathFor('foo/bar')).reply(200, null);
+      httpMock
+        .scope(baseUrl)
+        .get(pathFor('foo/bar'))
+        .reply(200, null as never);
```

Comment:
null isn't allowed / valid
```suggestion
        .reply(200);
```

Use this to force got to throw an error
```suggestion
        .reply(200, '{');
```